Changes.md

```diff
@@ -1,6 +1,11 @@
 1.6.x.x (relative to 1.6.3.0)
 =======
 
+Features
+--------
+
+- DataWindowQuery : Added a new node for querying image data windows.
```

Comment:
As this was requested in #5637, we should be including that here and in the commit message.